opa/data/ir_ui_menu.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <menuitem name="Open Academy" id="menu_main" sequence="65">
```

Comment:
Menu items are not usually located in the data dir, but in the views, usually in the model's view file or, for more general menus, in `menuitems.xml`

Reply:
This was my suggestion because the model of menu items is ir.ui.menu model.